components/lua/lua.c

```diff
@@ -39,7 +39,7 @@ static void lstop (lua_State *L, lua_Debug *ar) {
 
 
 static void laction (int i) {
-  // signal(i, SIG_DFL); 
```

Comment:
My editor is configured to strip trailing whitespace. I can rework this to have only the added line if that would be preferred.